budgetportal/tests/test_pages.py

```diff
@@ -155,7 +155,7 @@ def test_faq_page(self):
         c = Client()
         response = c.get('/faq')
 
-        self.assertContains(response, 'When is the budget data updated?')
+        self.assertContains(response, 'Frequently Asked Questions (FAQ)')
```

Comment:
please load the faq fixture in this test class and check chat one of the FAQ titles and a snippet of its answer is in the body

Reply:
fixture added and test updated:

https://github.com/vulekamali/datamanager/pull/179/commits/578d822f00f28fd3ca9503ec32e6469f0c36526e